### PR TITLE
networkd: RPDB rule - add support to configure inverted rule.

### LINKED ISSUE #10706: networkd: RoutingPolicyRule fib flags
**Is your feature request related to a problem? Please describe.**
I need to setup an inverted rule, as in `ip rule add not ...`,
e.g. ```ip rule add not to $vpn_server_ip table $forward_through_vpn```

**Describe the solution you'd like**
Add "Not" field or "Flags" field, similar to https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/tree/ip/iprule.c#n715
I'd have done it myself, but I'm neither familiar with netlink or networkd codebase, nor yet sure how much code needs to be updated (it seems there are a couple places where RoutingPolicyRule gets copied member-by-member or its fields passed as plain function parameters -- don't recall exactly, but sure saw smth like that).

**Describe alternatives you've considered**
Ugh, adding that rule manually on each boot

P.S. Am I right thinking that libsystemd/sd-netlink is basically a re-implementation of some part of libnl? If so, why wasn't the latter used?

### PR DESCRIPTION
**Pull request by ssahani**
Closes #10706

### CODE REVIEW

File: src/network/networkd-routing-policy-rule.c [hunk 1]
```diff
@@ -595,6 +595,12 @@ int routing_policy_rule_configure(RoutingPolicyRule *rule, Link *link, sd_netlin
                         return log_error_errno(r, "Could not append FRA_DPORT_RANGE attribute: %m");
         }
 
+        if (rule->invert_rule > 0) {
```
Comment (yuwata) [inline src/network/networkd-routing-policy-rule.c]: `rule->invert_rule` is boolean. So, please drop `> 0`.

Comment (ssahani) [reply src/network/networkd-routing-policy-rule.c]: ouch

**Comment by ssahani** [comment]
added thanks @yuwata for reviewing.

**Comment by yuwata** [comment]
Thanks. Now LGTM. I will test this later.

**Pull request merged**